tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -445,12 +445,13 @@ private static void InitRemoteConfigurationManagement(Tracer tracer)
         {
             // Service Name must be lowercase, otherwise the agent will not be able to find the service
             var serviceName = TraceUtil.NormalizeTag(tracer.Settings.ServiceName ?? tracer.DefaultServiceName);
+            var configurationManager = tracer.TracerManager.RemoteConfigurationManager;
```

Comment:
Do we need this line? It looks like we may be able to remove it

Reply:
Good point

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/4202/commits/998b23e833715193fa29777aabe9a4a3b5c17dd8